src/client/Microsoft.Identity.Client/OAuth2/MsalTokenResponse.cs

```diff
@@ -191,35 +191,36 @@ public void Log(ICoreLogger logger, LogLevel logLevel)
                 StringBuilder withPii = new StringBuilder();
                 StringBuilder withoutPii = new StringBuilder();
 
-                withPii.AppendLine("==MsalTokenResponse==");
-                withoutPii.AppendLine("==MsalTokenResponse==");
-
-                withPii.AppendLine($"Error: {Error} ErrorDescription: {ErrorDescription}");
-                withoutPii.AppendLine($"Error: {Error} ErrorDescription: {ErrorDescription}");
+                withPii.AppendLine($"{Environment.NewLine}[MsalTokenResponse]");
+                withPii.AppendLine($"Error: {Error}");
+                withPii.AppendLine($"ErrorDescription: {ErrorDescription}");
                 withPii.AppendLine($"Scopes: {Scope} ");
+                withPii.AppendLine($"ExpiresIn: {ExpiresIn}");
+                withPii.AppendLine($"RefreshIn: {RefreshIn}");
+                withPii.AppendLine($"AccessToken: {!string.IsNullOrEmpty(AccessToken)}");
```

Comment:
Maybe we should rather write "AccessTokenReturned" ? as this is just the presence of the AT?